platform/wayland/os-compatibility.c

```diff
@@ -21,6 +21,8 @@
  * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
+ *
+ * SPDX-License-Identifier: MIT
```

Comment:
ditto.

Reply:
Same here, I'll revert this before landing.

Reply:
I will also revert the change in `platform/gtk4/cog-gtk-settings-{dialog,cell-renderer-variant}.[hc]`, which were picked from the WebKit MiniBrowser.

Reply:
Oh, and `platform/drm/kms.[hc]`, which come from libdrm IIRC.